### PR TITLE
Add some resource

### PR DESCRIPTION
**Pull request by SuliacLEGUILLOU**
Hello!
I added two little resource I find very useful.

I hesitated to create a "Math" category for the first one but I don't know if it's fine to have a category for a single link so I added it to general. It may be an interesting idea in the future to gather a bit more resource on the topic to create a dedicated category.

Thank you for reading and have a nice day!

### CODE REVIEW

File: store.json [hunk 1]
```diff
@@ -382,7 +382,12 @@
         "title": "The A11Y Checklist",
         "desc": "This checklist uses the The Web Content Accessibility Guidelines (WCAG) as a reference point. The WCAG is a shared standard for web content accessibility for individuals, organizations, and governments.",
         "url": "https://a11yproject.com/checklist/"
-      }
+	    },
+	    {
+        "title": "Math ∩ Programming",
+        "desc": "Set of tutorial and article about various Math` tools and how to use them in computer science.",
```
Comment (lostdesign) [inline store.json]: Please get rid of the backtick after math.

Comment (SuliacLEGUILLOU) [reply store.json]: Typo fixed!

**Comment by lostdesign** [comment]
Thanks, I guess in order to push to store, we'll merge this after the switch. @S3B4S what do you say?

**Comment by SuliacLEGUILLOU** [comment]
@lostdesign not sure of what you mean here, but i'll keep an eye on that PR and make sure it can be merge at any time

**Comment by S3B4S** [comment]
@SuliacLEGUILLOU We're changing `store.json` into multiple `<category>.json` files, see #91. That's what @lostdesign means. :)

I'm surprised we didn't have https://internetingishard.com/ already, it's a great resource. Though I'd prefer a link to the homepage instead of directly to the first lesson.

**Comment by SuliacLEGUILLOU** [comment]
Sounds like a nice idea indeed!
I'll keep an eye on that PR and adapt it once you are done with the split.

Regarding of the lesson/homepage thing, I felt like a link to the direct content was more useful as I prefer getting straight to the point, but I'll edit it to indicate the homepage when I'll work on that PR again.

**Comment by S3B4S** [comment]
@SuliacLEGUILLOU #91 has been merged! So if you could resolve the conflicts and edit as above discussed that'd be great 👍

**Comment by lostdesign** [comment]
Also, please reopen this into master.

**Pull request closed without merging**